src/components/index.ts

```diff
@@ -0,0 +1,16 @@
+import { CustomButton } from './CustomButton/CustomButton';
+import { Logo } from './Logo/Logo';
+import { CustomAppBar } from './CustomAppBar/CustomAppBar';
+import { ActiveNote } from './ActiveNote/ActiveNote';
+import { EditMenuContainer } from './EditMenu/EditMenuContainer';
+import { DisplayedNote } from './DisplayedNote/DisplayedNote';
+import { CreateNoteContainer } from './CreateNote/CreateNoteContainer';
+export {
```

Comment:
and also it's better aproach to add index.ts file inside components folder as you did, but containing the following, so you don't need to import components and then export them again

export { default as CustomButton } from './CustomButton';
export { default as Logo } from './Logo';